charts/sshpiper/templates/secret.yaml

```diff
@@ -1,11 +1,13 @@
 apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "sshpiper.fullname" . }}-server-key
+  labels:
+    {{- include "sshpiper.labels" . | nindent 4 }}
+type: Opaque
 data:
   {{- if .Values.sshpiper.ssh_host_key }}
-  server_key: {{ .Values.sshpiper.ssh_host_key }}
```

Comment:
helm --set sshpiper.ssh_host_key= multiline?
here i am assuming the value is base64 because multiline bash param is not reasonable

Reply:
That's valid point, but IMHO very edge case.

- he can still pass non-multiline base64 and decode on the fly `--set "sshpiper.ssh_host_key=$(base64 -d <<< "aGVsbG8K")"` if he really wants
- not many people would want to pass such big arguments **interactivelly** and for non-interactive usage in scripts, it's not a concern at all
- just for completeness, passing secrets over arguments is not a good idea anyway
- if he reads from values/GitOps he probably more appreciates having there unobfuscated key to see what really is deployed,
- ... but mainly, from original name (`sshpiper.ssh_host_key`) it was not clear that it expects base64

So just from my personal perspective, it's not a case Helm chart should cover.

But if you insist, I would propose to
- at least rename the variable to `sshpiper.ssh_host_key_b64`,
- or (ehm...) I can introduce 2 variables (`ssh_host_key` and `ssh_host_key_b64`)

Reply:
lets make it `ssh_host_key_base64`?

Reply:
I have force pushed a change with `ssh_host_key_base64`.